libs/remix-ui/home-tab/src/lib/components/homeTabFile.tsx

```diff
@@ -146,16 +93,16 @@ function HomeTabFile({ plugin }: HomeTabFileProps) {
     await plugin.call('filePanel', 'switchToWorkspace', { name: wName, isLocalHost: false })
     await plugin.call('filePanel', 'switchToWorkspace', { name: wName, isLocalHost: false }) // calling once is not working.
     const content = `// SPDX-License-Identifier: MIT
-pragma solidity >=0.6.12 <0.9.0;
-
-contract HelloWorld {
-  /**
-   * @dev Prints Hello World string
-   */
-  function print() public pure returns (string memory) {
-    return "Hello World!";
-  }
-}
+        pragma solidity >=0.6.12 <0.9.0;
```

Comment:
this will be wrong in editor

Reply:
I just pasted the code into the contract and it worked without any errors. Maybe you are referring to something else?

Reply:
indentation will be wrong in remix editor